EPRN.Portal/Views/Prns/Action.cshtml

```diff
@@ -11,21 +11,20 @@
 
 <partial name="_ValidationSummary" />
 <form method="post">
-    <input type="hidden" asp-for="JourneyId" />
     <radios asp-for="DoWithPRN" asp-title="@ActionPRNResources.Heading">
         <p class="govuk-error-message">
             <span asp-validation-for="DoWithPRN"></span>
         </p>
 
         <div class="govuk-radios__item">
-            <input class="govuk-radios__input" type="radio" asp-for="DoWithPRN" value="@ActionPRN.SendItTo" aria-describedby="@ActionPRNResources.SendItTo">
+            <input class="govuk-radios__input" type="radio" asp-for="DoWithPRN" value="@PrnStatus.Complete" aria-describedby="@ActionPRNResources.SendItTo">
```

Comment:
Are we 100% this view Completes the PRN? I'm not convinced

Reply:
This has been updated.